mkdocs/docs/contributing/overview.md

```diff
@@ -23,19 +23,19 @@ bug report and let us know!
 We welcome many different types of contributions including:
 
 * [Iter8 documentation / tutorials](../tutorials)
```

Comment:
Personally, I don't like spaces before and after slashes. I think this happens a number of times in our docs so maybe we should decide on what to do in the future.